docs/getting-started.rst

```diff
@@ -105,56 +128,39 @@ properties of each retrieved Device object.
 
 Calling the SDK Directly
 ------------------------
+
 Now we'll repeat this example, but using the Python command line directly without a script.
 
-Access your Python interpreter with the ``python`` command (or ``python3`` if required) and type:
+Access your Python interpreter with the ``python`` command (or ``python3`` if required) and type::
 
->>> from cbc_sdk.rest_api import CBCloudAPI
->>> from cbc_sdk.platform import Device
->>> cb = CBCloudAPI(profile='default')
+    >>> from cbc_sdk.rest_api import CBCloudAPI
+    >>> from cbc_sdk.platform import Device
+    >>> cb = CBCloudAPI(profile='default')
 
 This imports the necessary classes and creates an instance of the base ``CBCloudAPI`` object.  By default, the file
 credentials provider is used. We set it to use the ``default`` profile in your ``credentials.cbc`` file, which you
 set up earlier.
 
-**N.B.:** On Windows, a security warning message will be generated about file access to CBC SDK credentials being
+.. note:: On Windows, a security warning message will be generated about file access to CBC SDK credentials being
 inherently insecure.
 
->>> query = cb.select(Device).where('1')
+This creates a query object that searches for all devices::
 
-This creates a query object that searches for all devices (the '1' causes all devices to be matched, as in SQL).
+    >>> query = cb.select(Device)
 
->>> devices = list(query)
+For convenience, we load the entirety of the query results into an in-memory list::
 
-For convenience, we load the entirety of the query results into an in-memory list.
-
->>> for device in devices:
-...     print(device.id, device.name, device.last_internal_ip_address, device.last_contact_time)
-...
+    >>> devices = list(query)
 
 Using a simple ``for`` loop, we print out the ID, host name, internal IP address, and last contact time from each
 returned device.  Note that the contents of the list are ``Device`` objects, not dictionaries, so we access individual
-properties with the ``object.property_name`` syntax, rather than ``object['property_name']``.
-
-Setting the User-Agent
-----------------------
-
-The SDK supports custom User-Agent's, which allow you to identify yourself when using the SDK to make API calls.
-The credential parameter ``integration_name`` is used for this. If you use a file to authenticate the SDK, this is
-how you could identify yourself:
+properties with the ``object.property_name`` syntax, rather than ``object['property_name']``::
 
-::
-
-  [default]
-  url=http://example.com
-  token=ABCDEFGHIJKLMNOPQRSTUVWX/12345678
-  org_key=A1B2C3D4
-  integration_name=MyScript/0.9.0
-
-See the :doc:`authentication` documentation for more information about credentials.
+    >>> for device in devices:
+    ...     print(device.id, device.name, device.last_internal_ip_address, device.last_contact_time)
+    ...
 
 Next Steps
```

Comment:
Can we reference the searching guide specifically as that would be the most logical next step? We should still reference the guides homepage too for more specific use cases

Reply:
Searching is an important operation in the SDK, and I added a note immediately after this example with the link you want.